CMakeSettings.json

```diff
@@ -1,22 +1,34 @@
 ﻿{
-  "configurations": [
-    {
-      "name": "x64-Debug",
-      "generator": "Ninja",
-      "configurationType": "Debug",
-      "inheritEnvironments": [ "msvc_x64_x64" ],
-      "buildRoot": "${projectDir}\\out\\build\\${name}",
-      "installRoot": "${projectDir}\\out\\install\\${name}",
-      "cmakeCommandArgs": "",
-      "buildCommandArgs": "-v",
-      "ctestCommandArgs": "",
-      "variables": [
+    "configurations": [
```

Comment:
Why is the indentation of the JSON changing in this PR? I think when I update this file in VS, it reverts back to 2 spaces.

Reply:
Clang doesn't format these, VS does.  My VS was set to 4 and I didn't notice.  Please put it back to two when you commit your updates.